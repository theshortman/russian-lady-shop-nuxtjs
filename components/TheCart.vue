<template>
  <div class="list-group mb-3">
    <NuxtLink to="/cart" class="list-group-item text-reset">
      <div class="list-group-item-heading">
        <span class="text-danger">КОРЗИНА</span>
      </div>

      <div class="list-group-item-text">Пока пусто!</div>
    </NuxtLink>
  </div>
</template>