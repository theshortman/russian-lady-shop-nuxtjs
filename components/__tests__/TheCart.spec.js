import { shallowMount, RouterLinkStub } from '@vue/test-utils'
import TheCart from '../TheCart'

describe('TheCart.vue', () => {
    test('renders correctly', () => {
        const wrapper = shallowMount(TheCart, {
            stubs: {
                NuxtLink: RouterLinkStub
            }
        })
        
        expect(wrapper.html()).toMatchSnapshot()
    })
})