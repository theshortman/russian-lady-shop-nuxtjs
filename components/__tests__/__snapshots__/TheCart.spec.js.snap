// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheCart.vue renders correctly 1`] = `
"<div class=\\"list-group mb-3\\"><a class=\\"list-group-item text-reset\\">
    <div class=\\"list-group-item-heading\\"><span class=\\"text-danger\\">КОРЗИНА</span></div>
    <div class=\\"list-group-item-text\\">Пока пусто!</div>
  </a></div>"
`;
